pkg/cloudprovider/aws/fake/filesystem.go

```diff
@@ -0,0 +1,20 @@
+package fake
```

Comment:
Thoughts on moving this to pkg/test/fake since it's not aws specific?

Reply:
Will do